tidb-cloud/data-service-api-key.md

```diff
@@ -0,0 +1,51 @@
+---
+title: API Key in Data Service
+summary: Learn how to create, edit, and delete an API key for a Data App.
+---
+
+# API Key in Data Service
+
+The TiDB Cloud Data API uses [HTTP Digest Authentication](https://en.wikipedia.org/wiki/Digest_access_authentication). It protects your private key from being sent over the network. For more details about HTTP Digest Authentication, refer to the [IETF RFC](https://datatracker.ietf.org/doc/html/rfc7616).
+
+## API key overview
+
+- The API key contains a public key and a private key, which act as the username and password required in the HTTP Digest Authentication. The private key only displays upon the key creation.
```

Comment:
```suggestion
- An API key contains a public key and a private key, which act as the username and password required in the HTTP Digest Authentication. The private key is only displayed upon the key creation.
```

Reply:
`Display` is a transitive verb; therefore, it requires an object.
https://developers.google.cn/style/word-list#letter-d